reviews/review-priorities.md

```diff
@@ -16,12 +16,16 @@ Classifications are done through labels, e.g.: priority-high.
 
 Please note that these are the priorities used by the core team. External contributors are welcome to review PRs according to their own priorities, and such review activity will help highlight PRs which should get attention from the core team.
 
+## priority-very-high
+
+Same criteria as priority-high, below, but needed for the Mbed TLS teams' most important commitments.
+
 
 ## priority-high
 
-This classification is used for PRs which are typically one of:
+This classification is used for PRs which have a target date for merging - typically one of:
 
-- Security (or other high-priority) bug fixes
+- Security (or other high-priority) bug fixes needed in the next release
```

Comment:
Security fixes needed in the next release are mostly `priority-very-high` as they block releases, force releases and often have a hard deadline for public disclosure. Maybe we could mention them as an example in the very high category.

Reply:
They are only `priority-very-high` if the release itself is a diamond commitment, which usually it isn't (although maybe that should change).